app/controllers/returns/ExportedPlasticPackagingWeightController.scala

```diff
@@ -69,7 +70,7 @@ class ExportedPlasticPackagingWeightController @Inject()(
             Future.successful(handleTotalPlasticCalculationError(mode, formWithErrors)),
           value =>
             request.userAnswers
-              .setOrFail(ExportedPlasticPackagingWeightPage, value)
+              .setOrFail(ExportedPlasticPackagingWeightPage, value, isCleanUp(mode))
```

Comment:
this maybe need a better way/rethinking, but for the moment this is the quickest way to fix it

Reply:
Ok. Starting with something that works seems like a good way to do it!